packages/backend/src/payment-method/local/service.ts

```diff
@@ -117,17 +121,20 @@ async function getQuote(
     exchangeRate = convertResults.scaledExchangeRate
   } else if (receiveAmount) {
     receiveAmountValue = receiveAmount.value
-    const convertResults = await convert({
-      destinationAmount: receiveAmountValue,
-      sourceAsset: {
-        code: walletAddress.asset.code,
-        scale: walletAddress.asset.scale
+    const convertResults = await convert(
+      {
+        destinationAmount: receiveAmountValue,
+        sourceAsset: {
+          code: walletAddress.asset.code,
+          scale: walletAddress.asset.scale
+        },
+        destinationAsset: {
+          code: receiveAmount.assetCode,
+          scale: receiveAmount.assetScale
+        }
       },
-      destinationAsset: {
-        code: receiveAmount.assetCode,
-        scale: receiveAmount.assetScale
-      }
-    })
+      options.walletAddress.tenantId
```

Comment:
There's an edge case here: the sending wallet's tenant could have only configured a single asset (and not having any rates URL set as a result), but the receiver's tenant wallet is in a different asset code (who might also be only supporting a single asset). 

I'm not sure there is a straightforward "coding" solution here but a business decision one: _someone_ would need to know in this case what the correct FX is, maybe it is the operator.